documentation/Apple Unified Logging and Activity Tracing formats.asciidoc

```diff
@@ -547,15 +571,64 @@ opendirectoryd (build 483.200) launched...
 
 ....
 
+=== Oversize chunk
+
+The Oversize chunk is variable in size and consists of:
+
+[cols="1,1,1,5",options="header"]
+|===
+| Offset | Size | Value | Description
+4+| _Chunk header (tracev3_chunk_preamble)_
+| 0 | 4 | 0x6002 | Chunk tag (tag)
+| 4 | 4 | | Chunk sub tag (subtag)
+| 8 | 8 | | Chunk data size (length)
+4+| _Chunk data (tracev3_chunk_oversize)_
+| 16 | 8 | | First number in proc_id #@#
+| 24 | 4 | | Second number in proc_id #@#
+| 28 | 1 | | TTL
+| 29 | 3 | 0 | *Unused/Reserved*
+| 32 | 8 | | Continuous time
+| 40 | 4 | | Data reference index
+| 44 | 4 | | Size of data (to follow)
+| 48 | ... | | Data
+|===
+Oversize chunks consist of data that is too large to fit into a single log record, hence it is stored in a seperate record here and referenced (by log entires in firehose).
+
+=== Statedump chunk
+
+The Statedump chunk is variable in size and consists of:
+
+[cols="1,1,1,5",options="header"]
+|===
+| Offset | Size | Value | Description
+4+| _Chunk header (tracev3_chunk_preamble)_
+| 0 | 4 | 0x6003 | Chunk tag (tag)
+| 4 | 4 | | Chunk sub tag (subtag)
+| 8 | 8 | | Chunk data size (length)
+4+| _Chunk data (tracev3_chunk_statedump)_
+| 16 | 8 | | First number in proc_id #@#
+| 24 | 4 | | Second number in proc_id #@#
+| 28 | 1 | | TTL
+| 29 | 3 | 0 | *Unused/Reserved*
+| 32 | 8 | | Continuous time
+| 40 | 8 | | Activity Id
+| 48 | 16 | | UUID 
+| 64 | 4 | | Data Type (1=plist, 3=data)
+| 68 | 4 | | Size of data (to follow)
```

Comment:
The '(to follow)' may be misinterpreted here, so please remove it.

Reply:
You can ignore this for now, I will rewrite the entries from offset 64 to 264, with better descriptions.

Reply:
ack